app/src/main/java/org/fossasia/openevent/general/event/EventService.kt

```diff
@@ -21,6 +21,20 @@ class EventService(private val eventApi: EventApi, private val eventDao: EventDa
         }
     }
 
+    fun getSearchEvents(eventName: String): Flowable<List<Event>> {
+        val eventsFlowable = eventDao.getSearchEvents()
+        return eventsFlowable.switchMap {
+            eventDao.deleteAll()
```

Comment:
Why delete all while searching?

Reply:
So that every time I search something,the previous results stored in db won't be shown. I wasn't able to come up with a better solution than this

Reply:
Why query the database at all?

Reply:
Thanks. This will solve all the above problem :smile: